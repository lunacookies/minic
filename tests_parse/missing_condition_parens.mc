func main {
	if (0
		set x = 10;
	if 0
		set x = 10;
	if 0)
		set x = 10;
	while (0
		set x = 10;
	while 0
		set x = 10;
	while 0)
		set x = 10;
}
