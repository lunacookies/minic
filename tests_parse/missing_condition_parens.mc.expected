func main {
	if 0
		set x = 10
	
	if 0
		set x = 10
	
	if 0
		set x = 10
	
	while 0
		set x = 10
	while 0
		set x = 10
	while 0
		set x = 10
}
tests_parse:18..19: error: missing “)”
tests_parse:36..37: error: missing “(”
tests_parse:38..39: error: missing “)”
tests_parse:56..57: error: missing “(”
tests_parse:83..84: error: missing “)”
tests_parse:104..105: error: missing “(”
tests_parse:106..107: error: missing “)”
tests_parse:127..128: error: missing “(”
